editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/internal/l10n/messages.properties

```diff
@@ -70,6 +70,8 @@ autoLayoutActionDescription=Automatic layout of the diagram
 autoLayoutActionTooltip=Does an automatic layout of the diagram
 beanClassExistsValidatorErrorBeanClassMandatory=Bean class name is mandatory.
 beanClassExistsValidatorErrorBeanClassMustExist=Bean class must exist in the Fuse project.
+beanConfigUtilNoMethodsAvailable=No methods available.
```

Comment:
Essentially I think we need to make our own dialog that offers a bit more feedback to the user.